server/src/Utopia/Web/Endpoints.hs

```diff
@@ -277,7 +277,7 @@ injectIntoPage toInject@(_, _, _, _, editorScriptTags) (TagComment "editorScript
 injectIntoPage toInject (firstTag : remainder) = firstTag : injectIntoPage toInject remainder
 injectIntoPage _ [] = []
 
-renderPageWithMetadata :: Maybe ProjectIdWithSuffix -> Maybe ProjectMetadata -> Maybe DB.DecodedProject -> Maybe Text -> Text -> ServerMonad H.Html
+renderPageWithMetadata :: Maybe ProjectIdWithSuffix -> Maybe ProjectMetadata -> Maybe DB.DecodedProject -> Maybe Text -> Text -> ServerMonad ProjectPageResponse
```

Comment:
The new `ProjectPageResponse` type is so that the return values of these functions can include the new headers required for `crossOriginIsolated`